.github/workflows/label-schema.yml

```diff
@@ -12,4 +12,5 @@ jobs:
       continue-on-error: true
       env:
         SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_SCHEMA_CHANGE }}
-        SLACK_FOOTER: ${{ github.event.pull_request.title }}
+        SLACK_MESSAGE: ${{ github.event.pull_request.title }}
+        SLACK_FOOTER: "<${{ github.server_url }}/${{ github.repository }}/pull/${{ github.event.pull_request.number }}>"
```

Comment:
Not a blocker, but I'm curious if the pull request event exposes the url directly so it doesn't need to be built up from substrings.

https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request

Reply:
Thank you for the review. I couldn't find the field for the URL directly unfortunately.